### PR TITLE
[DO NOT MERGE] chore: june milestone

### PR DESCRIPTION
**Pull request by dhmlau**
Add June milestone. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/_jun_milestone_.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Finish Q2 release
+
+## Goals
+
+- [ ] Clean up extension template and docs #5336
+- [ ] Failed to exclude properties by setting false in `fields` filter via url
+      #4992
+- [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+- [ ] Integration with TypeORM #4853
+
```
Comment (dhmlau) [inline docs/_jun_milestone_.md:14]: @agnes512, would it make sense to make a June goal to get the hasManyThrough original PR to `master`?

File: docs/_jun_milestone_.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Documentation improvement
+
+  - [ ] [Docs] MongoDB connector docs should cover LB4 content #4879
+  - [ ] [Docs] MySQL connector docs should cover LB4 content #4878
+  - [ ] [Docs] Oracle connector docs should cover LB4 content #4880
+  - [ ] [Docs] Add "project layout reference" page #4847
+
+- [ ] [Workshop] From API to UI: LoopBack with Angular with client generator
+      #4849
```
Comment (dhmlau) [inline docs/_jun_milestone_.md:25]: (Proposal) PRs to review:
- [ ] [RFC] Enable parallel mocha testing #5011 (IIUC, waiting for the next mocha release)
- [ ] feat(http-server): adds http/2 support #4989 (probably need more review time?)
- [ ] feat(graphql): add an extension for type-graphql integration #5545

Comment (jannyHou) [reply docs/_jun_milestone_.md:25]: @dhmlau 
> feat(http-server): adds http/2 support #4989 (probably need more review time?)

This PR is waiting for Express's support of http2, please see discussion https://github.com/strongloop/loopback-next/pull/4989#issuecomment-625380889 and https://github.com/strongloop/loopback-next/pull/4989#issuecomment-633346475.

> RFC] Enable parallel mocha testing #5011 (IIUC, waiting for the next mocha release)

Yep, it's waiting for mocha release.

**Comment by dhmlau** [comment]
Closing this PR as https://github.com/strongloop/loopback-next/issues/5607 is opened.

**Pull request closed without merging**